app/templates/views/organisations/add-gp-organisation.html

```diff
@@ -6,16 +6,12 @@
 {% from "components/form.html" import form_wrapper %}
 {% from "components/uk_components/back-link/macro.njk" import govukBackLink %}
 
-{% block per_page_title %}
-  Accept our data sharing and financial agreement
-{% endblock %}
```

Comment:
This makes me think that this whole page is agreement related. However, the stuff specific to `GP` and `NHS` aren't active and will eventually get removed as UK-specific, so I don't think we need to worry about that